src/main/kotlin/com/workos/organizations/OrganizationsApi.kt

```diff
@@ -75,15 +76,57 @@ class OrganizationsApi(private val workos: WorkOS) {
     }
   }
 
+  @JsonInclude(Include.NON_NULL)
+  class CreateOrganizationRequestOptions constructor(
+    val idempotencyKey: String? = null,
+  ) {
+    /**
+     * Builder class for [CreateOrganizationRequestOptions].
+     */
+    class CreateOrganizationRequestOptionsBuilder {
+      private var idempotencyKey: String? = null
+
+      /**
+       * Sets the idempotencyKey.
+       */
+      fun idempotencyKey(value: String) = apply { idempotencyKey = value }
+
+      /**
+       * Creates a [CreateOrganizationRequestOptions] with the given builder parameters.
+       */
+      fun build(): CreateOrganizationRequestOptions {
+        return CreateOrganizationRequestOptions(idempotencyKey)
```

Comment:
Since it's currently the only reason for passing the request options, maybe we can do the same validation we are doing here: https://github.com/workos/workos-kotlin/blob/main/src/main/kotlin/com/workos/auditlogs/AuditLogsApi.kt#L188-L194

Reply:
I was wondering whether to do that @robframpton but wouldn't that imply that the Idempotency Key is required though and in this case we're keeping it as optional?

Reply:
I could see it both ways I suppose. On one hand it prevents developers from passing that object for no reason, however thinking on it now I could see someone doing the following.

```
val optionsBuilder = CreateOrganizationRequestOptions.builder();

if (someBoolean) {
  optionsBuilder.idempotencyKey("key");
}

workos.organizations.createOrganization(options, optionsBuilder.build());
```

Let's merge as is 👍🏻